python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -33,8 +33,8 @@
 
 
 class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
-    def _validateGalsimInterpolant(name: str) -> bool:  # noqa: N805
-        """A helper function to validate the GalSim interpolant at config time.
+    def validateGalsimInterpolant(name: str) -> bool:  # noqa: N805
```

Comment:
I still think this should be a private method (leading underscore) even though it is used in the tests.

Also, I don't know how this works as a static method without the `@staticmethod` decorator... or wait, it's because it's a local method, I guess, that just happens to work because it's at the class scope.

Recommendation: remove this from the class level, the check function should be outside the class (at line 33), and have a leading underscore.  The `noqa` can then be removed and the scope will make more sense.